Userland/Libraries/LibWeb/HTML/HTMLLinkElement.cpp

```diff
@@ -55,9 +55,17 @@ void HTMLLinkElement::inserted()
 
 void HTMLLinkElement::parse_attribute(FlyString const& name, String const& value)
 {
+    // https://html.spec.whatwg.org/multipage/links.html#rel-icon
```

Comment:
This now links to `rel-icon` It should probably link to the link types (which the comments also references)
https://html.spec.whatwg.org/multipage/links.html#linkTypes